Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -79,7 +79,7 @@ private Stream Stream
         {
             _uri = uri;
             _useTls = useTlsEncryption;
-            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(true);
+            await _client.ConnectAsync(uri.Host, uri.Port).ConfigureAwait(false);
```

Comment:
@cskardon Hello, I am looking at this code that you added before, but I do not understand why this await has to go back to its calling context. Do you still remember why configAwait has to be `true` here? Can I change it to `false`?